resources/views/external-link-confirm.blade.php

```diff
@@ -11,10 +11,20 @@
     x-data="{
         openModal() {
             Livewire.emit('openModal', '{{ $identifier }}')
+        },
+        redirect() {
+            window.location.href = '{{ $url }}'
+        },
+        hasDismissedCookie() {
+            if (document.cookie.match('dismiss_external_link_alert=true')) {
+                return true;
+            }
+
+            return false;
```

Comment:
You can cast the result to bool and return it.
```suggestion
            return !! document.cookie.match('dismiss_external_link_alert=true');
```